packages/api/src/schema/schema.yaml

```diff
@@ -658,6 +661,11 @@ components:
       required: []
       table: null
 
+    webhook-patch-payload:
+      $ref: "#/components/schemas/webhook"
+      required: []
+      table: null
```

Comment:
This will allow requests with any webhook fields to go through, they will just silently fail because we ignore them. So I think it's a better experience for the API user if you explicitly list the editable fields here, like on the `stream-patch-payload` (and make sure to have a `additionalProperties: false`).